src/main/java/frc/robot/subsystems/elevator/ElevatorIO.java

```diff
@@ -12,7 +12,8 @@ public static class ElevatorIOInputs {
     public double elevatorPositionMeters = 0.0;
     public double elevatorVelocityMetersPerSec = 0.0;
     public double elevatorAppliedVolts = 0.0;
-    public double[] elevatorCurrentAmps = new double[] {};
+    public double[] elevatorStatorCurrentAmps = new double[] {};
+    public double[] elevatorSupplyCurrentAmps = new double[] {};
```

Comment:
Like Kevin's -- not blocking -- why are we using an array for this? In other languages, this is represented as a tuple, which is an indicator for abstracting to a class. (also see below relating comment).

Reply:
especially since this is not being used for iteration.